### PR TITLE
Blocked direct access to Twig template files

### PR DESCRIPTION
**Pull request by evrifaessa**
We've migrated from .phtml to .html.twig long ago. This PR updates the server configuration files to block direct access to Twig template files with the new extension.

### CODE REVIEW

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
Also update nginx template

**Review by BelleNottelling — APPROVED** [review]

**Comment by evrifaessa** [comment]
@jaapmarcus done. Also updated all of them to return 404 instead of 403.

File: data/nginx.conf [hunk 1]
```diff
@@ -14,6 +14,10 @@
      
      include /etc/nginx/mime.types;
 
+     # Block access to sensitive files and return 404 to make it indistinguishable from a missing file
+     location ~ (\.htaccess$|\.htpasswd$|\.ini$|\.log$|\.sh$|\.inc$|\.bak$|\.twig$|\.sql$) {
```
Comment (jaapmarcus) [inline data/nginx.conf:18]: .htaccess and .htpassword is allready blocked by 
https://github.com/FOSSBilling/FOSSBilling/blob/e0cb529ca86351cfa80e1a34a563853d81a092eb/data/nginx.conf#L56-L58

How ever I suggest blocking  all hidden files by:

https://github.com/hestiacp/hestiacp/blob/e3e172d893f20c303941a1ab2d846eeb823a9179/install/deb/templates/web/nginx/php-fpm/default.tpl#L42-L45

Also this looks nicer:
 location ~* ^.+\.(ini|sh|inc|bak|twig|sql|)$ {

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

**Comment by evrifaessa** [comment]
@jaapmarcus I've changed the files according to your suggestions. Anything else?

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
No

**Pull request merged**